app/views/homepage/_promotion-slots.html.erb

```diff
@@ -5,37 +5,30 @@
 <section>
   <div class="govuk-grid-row" data-module="gem-track-click">
     <div class="govuk-grid-column-one-third">
-      <a href="/government/topical-events/cop26"
+      <a href="/coronavirus"
          aria-hidden="true"
          tabindex="-1"
          data-track-category="homepageClicked"
          data-track-action="promoImageLink"
-         data-track-label="/government/topical-events/cop26"
+         data-track-label="/coronavirus"
          data-track-value="1"
          data-track-dimension-index="29"
-         data-track-dimension="homepage/cop26.jpg">
-        <%= image_tag "homepage/cop26.jpg", {
+         data-track-dimension="homepage/covid-19-promo.png">
+        <%= image_tag "homepage/covid-19-promo.png", {
           alt: "",
           class: "home-promo__image",
           height: 407,
           loading: "lazy",
           width: 610,
           sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
-          srcset: {
-            "homepage/cop26.jpg": "610w",
-            "homepage/cop26-480.jpg": "480w",
-            "homepage/cop26-320.jpg": "320w",
-            "homepage/cop26-240.jpg": "240w",
-            "homepage/cop26-170.jpg": "170w",
-          },
```

Comment:
Could you also add the other image sizes here?  This is good for devices with smaller screens, which can download a smaller image.

Reply:
The [docs](https://docs.publishing.service.gov.uk/apps/frontend/update-homepage-promotion-slots.html#3-update-the-markup-and-phrases) suggest we only need the smaller images for photographs (or complex graphics) rather than simple graphics like this one. We also don't provide the `srcset` attributes for the other promo images on the homepage. Do you think this is wrong? I.e. should I update the docs and add smaller images for all three slots?

Reply:
Ok, if that's only for photos then we don't need to do it here.